### PR TITLE
Add prayer times settings

### PR DESCRIPTION
**Pull request by muhajirdev**
### Summary 
Add prayer times settings

### CODE REVIEW

File: src/utils/input.ts [hunk 1]
```diff
@@ -13,7 +14,7 @@ import TafsirInfo from 'types/TafsirInfo';
  */
 export const generateSelectOptions = (options: string[]): SelectOption[] =>
   options.map((option) => ({
-    label: capitalize(option),
+    label: startCase(option),
```
Comment (muhajirdev) [inline src/utils/input.ts:17]: startCase will convert "MuslimWordLeague" -> "Muslim Word League"

File: src/components/Navbar/SettingsDrawer/PrayerTimesSection.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  CalculationMethod,
+  Madhab,
+  selectCalculationMethod,
+  selectMadhab,
+  setCalculationMethod,
+  setMadhab,
+} from 'src/redux/slices/prayerTimes';
+import { generateSelectOptions } from 'src/utils/input';
+
+const calculationMethodOptions = generateSelectOptions([
```
Comment (tifa2UP) [inline src/components/Navbar/SettingsDrawer/PrayerTimesSection.tsx:17]: loop the enum

File: src/utils/input.ts (hunk 1 above)
Comment (tifa2UP) [reply src/utils/input.ts:17]: this should happen before passing the input

File: src/redux/slices/prayerTimes.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { RootState } from '../RootState';
+
+export enum CalculationMethod {
```
Comment (tifa2UP) [inline src/redux/slices/prayerTimes.ts:5]: this should persist across sessions

**Review by tifa2UP — APPROVED** [review]

**Pull request merged**